### PR TITLE
Dont let proxy check fail if registry access is disabled

### PR DESCRIPTION
**Pull request by nagilson**
The proxy library fails if registry access is disabled, wrap it in a try catch.
Bump version to 1.7.4.

### CODE REVIEW

File: vscode-dotnet-runtime-extension/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning].
 
 ## [Unreleased]
 
+## [1.7.4] - 2023-08-24
+
+Install using node.arch() instead of environment variable architecture from root terminal to prevent mismatching shell architecture with vs code architecture.
```
Comment (nagilson) [inline vscode-dotnet-runtime-extension/CHANGELOG.md:12]: The node.arch() part will be added in another pr.

**Review by baronfel — APPROVED** [review]
This LGTM - for the new event we should filter that out in the telemetry and not consider it a terminal error (for overall success calculations), right?
